src/content/app/tools/blast/blast-download/submissionDownload.ts

```diff
@@ -168,7 +168,7 @@ const createZipArchive = async (submission: EnrichedBlastSubmission) => {
   return zip;
 };
 
-const getNameForZipRoot = (submission: BlastSubmission) => {
+export const getNameForZipRoot = (submission: BlastSubmission) => {
```

Comment:
Since you reverted the commit that was using this function outside this module, it probably shouldn't be exported.